### ISSUE TITLE
Error when shift-clicking items in chat

### ISSUE THREAD

**Issue by ghost** [shirsig/aux-addon-TBC#10]
`Message: Interface\AddOns\aux-addon\aux-addon.lua:133: attempt to call field 'HandleModifiedItemClick' (a nil value)
Time: 01/06/19 19:14:07
Count: 4
Stack: [C]: in function `HandleModifiedItemClick'
Interface\AddOns\aux-addon\aux-addon.lua:133: in function <Interface\AddOns\aux-addon\aux-addon.lua:131>
(tail call): ?
[C]: in function `HandleModifiedItemClick'
Interface\FrameXML\PaperDollFrame.lua:1158: in function `PaperDollItemSlotButton_OnModifiedClick'
[string "*:OnClick"]:2: in function <[string "*:OnClick"]:1>
`

**Comment by shirsig** [comment]
Make sure u have the newest version of aux TBC, make a clean update by deleting the folder and downloading a new zip, delete the savedvariables while wow is not running, disable all other addons to see if theres a conflict.

If none of that works I cant help you at the moment, no time to maintain this fork, sorry.

**Comment by Minnona** [comment]
Can confirm it, got the latest version. Also happening with ctrl clicking items as well.

**Comment by Huuhailija** [comment]
Yup i confirm as of today download the addon, both shift+ ctrl clicking items present you the error above.

**Comment by Laekoth** [comment]
This is the only complaint i have about this addon :(

**Comment by shirsig** [comment]
does the newest update fix it?

**Comment by Laekoth** [comment]
I just downloaded it and checked - seems to work so far, thanks!

**Comment by Huuhailija** [comment]
Checked the latest version and it works fawlessly. THX!

**Comment by Minnona** [comment]
Fixed everything for me as well. Thank you!
